blog/2020-10-08-FacilityLocationProblemQuickstart.adoc

```diff
@@ -0,0 +1,98 @@
+= New quickstart for the facility location problem
+:imagesdir: 2020/10/08
```

Comment:
Doesn't this break rendering for "rake clean build"?
Interesting trick though if it works, as it means we can see the images in IntelliJ.

Reply:
Exactly. It makes the images show up in IDEA when editing. It doesn't break anything as far as I can tell.

Reply:
Perfect trick, we should use that for all blog posts then.